vividus-allure-adaptor/src/main/java/org/vividus/bdd/report/allure/AllureReportGenerator.java

```diff
@@ -111,6 +116,20 @@ private void generateReport()
         LOGGER.info("Allure report is successfully generated at: {}", reportDirectory.getAbsolutePath());
     }
 
+    private void writeCategoriesInfo() throws IOException
+    {
+        List<AllureCategory> categories = Arrays.asList(
+                new AllureCategory("Test defects", Collections.singletonList(Status.BROKEN)),
+                new AllureCategory("Product defects", Collections.singletonList(Status.FAILED)),
+                new AllureCategory("Known issue", Collections.singletonList(Status.UNKNOWN)));
+
+        try (BufferedWriter writer = Files.newBufferedWriter(resultsDirectory.toPath().resolve("categories.json")))
+        {
+            ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
since `writeExecutorInfo` creates `ObjectMapper` as well and `ObjectMapper` is thread-safe, I propose
- to create a new field: `ObjectMapper obejctMapper`
- to use it in both `writeCategoriesInfo` and `writeExecutorInfo`